src/components/CommentsSection.jsx

```diff
@@ -0,0 +1,21 @@
+export default () => {
+
+  return (
+    <section className="mb-5">
+      <div className="card bg-light">
+          <div className="card-body">
+              {/* <!-- Comment form--> */}
+              <form className="mb-4"><textarea className="form-control" rows="3" placeholder="Join the discussion and leave a comment!"></textarea></form>
+              {/* <!-- Single comment--> */}
```

Comment:
If you feel the need for comments you should probably extract it into a new component with a good name and this will give the context you are trying to give